catalog/ui/search-ui/standard/src/main/webapp/js/controllers/cesium.controller.js

```diff
@@ -293,6 +294,28 @@ define(['application',
                 }
             },
 
+            overlayImage: function (model) {
+                this.removeOverlay();
+
+                var coords = model.get('geometry').getPolygon();
+                var cartographics = _.map(coords, function(coord) {
+                    return Cesium.Cartographic.fromDegrees(coord.longitude, coord.latitude, coord.altitude);
+                });
+
+                var rectangle = Cesium.Rectangle.fromCartographicArray(cartographics);
+                var source = model.get('properties').get('source-id');
+                var id = model.get('properties').get('id');
+
+                this.overviewLayer = this.scene.imageryLayers.addImageryProvider(new Cesium.SingleTileImageryProvider({
+                    url: '/services/catalog/sources/' + source + '/' + id + '?transform=overview&align=true',
```

Comment:
This is good reasoning for the ActionProvider - we would actually be able to simply request this URL instead of constructing it ourselves in the UI (where it can get out of date).